crates/symbolicator/src/services/download/mod.rs

```diff
@@ -53,8 +55,10 @@ pub enum DownloadError {
     Sentry(#[from] sentry::SentryError),
     #[error("failed to fetch data from S3")]
     S3(#[from] s3::S3Error),
-    #[error("S3 error code: {1} (http status: {0})")]
-    S3WithCode(StatusCode, String),
+    #[error("aws-sdk: failed to fetch data from S3")]
+    S3SDK(#[from] aws_smithy_http::byte_stream::Error),
```

Comment:
I believe this should be `S3Sdk`?  Doesn't clippy complain about this?

Reply:
I don't see any complaints from Clippy. It's only used [here](https://github.com/josb/symbolicator/blob/task/aws-sdk-rust-migration/crates/symbolicator/src/services/download/s3.rs#L245). Do you want me to rename it?

Reply:
yeah, let's rename this

Reply:
Renamed.